cosmos/nibirudevnet-2.json

```diff
@@ -0,0 +1,51 @@
+{
+  "chainId": "nibiru-devnet-2",
```

Comment:
Hi. Could you verify the chainId on it?
`Chain identifier unmatched: (expected: nibiru-devnet, actual: nibirudevnet-2)`

Reply:
Updated